packages/ui/src/app/pages/Validators/ValidatorList.stories.tsx

```diff
@@ -0,0 +1,127 @@
+                { era: new BN(692), eraReward: new BN(1234560000) },
+                { era: new BN(693), eraReward: new BN(1234560000) },
+                { era: new BN(694), eraReward: new BN(1234560000) },
+                { era: new BN(695), eraReward: new BN(1234560000) },
+                { era: new BN(696), eraReward: new BN(1234560000) },
+                { era: new BN(697), eraReward: new BN(1234560000) },
+                { era: new BN(698), eraReward: new BN(1234560000) },
+                { era: new BN(699), eraReward: new BN(1234560000) },
+                { era: new BN(700), eraReward: new BN(1234560000) },
+              ],
+            },
+          },
+          query: {
+            balances: {
+              totalIssuance: joy(1000000),
+            },
+            timestamp: {
+              now: Date.now(),
+            },
+            session: {
+              validators: [
+                'j4RLnWh3DWgc9u4CMprqxfBhq3kthXhvZDmnpjEtETFVm446D',
+                'j4RbTjvPyaufVVoxVGk5vEKHma1k7j5ZAQCaAL9qMKQWKAswW',
+                'j4Rc8VUXGYAx7FNbVZBFU72rQw3GaCuG2AkrUQWnWTh5SpemP',
+                'j4Rh1cHtZFAQYGh7Y8RZwoXbkAPtZN46FmuYpKNiR3P2Dc2oz',
+                'j4RjraznxDKae1aGL2L2xzXPSf8qCjFbjuw9sPWkoiy1UqWCa',
+                'j4RuqkJ2Xqf3NTVRYBUqgbatKVZ31mbK59fWnq4ZzfZvhbhbN',
+                'j4RxTMa1QVucodYPfQGA2JrHxZP944dfJ8qdDDYKU4QbJCWNP',
+                'j4Rxkb1w9yB6WXroB2npKjRJJxwxbD8JjSQwMZFB31cf5aZAJ',
+                'j4RyLBbSUBvipuQLkjLyUGeFWEzmrnfYdpteDa2gYNoM13qEg',
+                'j4S998Thq5kQHyurofh8QfHrcFN2c1T19gTdMGUVVx5EHKgky',
+              ],
+            },
+            staking: {
+              activeEra: {
+                unwrap: () => ({
+                  index: new BN(700),
+                  start: Date.now() - 5400000,
+                }),
+              },
+              counterForValidators: new BN(12),
+              counterForNominators: new BN(20),
+              erasRewardPoints: () => {
+                return of({
+                  total: new BN(18000),
+                  individuals: {
+                    j4RLnWh3DWgc9u4CMprqxfBhq3kthXhvZDmnpjEtETFVm446D: '180',
+                    j4RbTjvPyaufVVoxVGk5vEKHma1k7j5ZAQCaAL9qMKQWKAswW: '200',
+                    j4Rc8VUXGYAx7FNbVZBFU72rQw3GaCuG2AkrUQWnWTh5SpemP: '280',
+                    j4Rh1cHtZFAQYGh7Y8RZwoXbkAPtZN46FmuYpKNiR3P2Dc2oz: '200',
+                    j4RjraznxDKae1aGL2L2xzXPSf8qCjFbjuw9sPWkoiy1UqWCa: '160',
+                    j4RuqkJ2Xqf3NTVRYBUqgbatKVZ31mbK59fWnq4ZzfZvhbhbN: '180',
+                    j4RxTMa1QVucodYPfQGA2JrHxZP944dfJ8qdDDYKU4QbJCWNP: '140',
+                    j4Rxkb1w9yB6WXroB2npKjRJJxwxbD8JjSQwMZFB31cf5aZAJ: '160',
+                    j4RyLBbSUBvipuQLkjLyUGeFWEzmrnfYdpteDa2gYNoM13qEg: '160',
+                    j4S998Thq5kQHyurofh8QfHrcFN2c1T19gTdMGUVVx5EHKgky: '220',
+                  },
+                })
+              },
```

Comment:
The decorator also takes of wrapping values in function returning an observer. So this should work:
```suggestion
              erasRewardPoints: {
                  total: joy('0.0018'),
                  individuals: {
                    j4RLnWh3DWgc9u4CMprqxfBhq3kthXhvZDmnpjEtETFVm446D: '180',
                    j4RbTjvPyaufVVoxVGk5vEKHma1k7j5ZAQCaAL9qMKQWKAswW: '200',
                    j4Rc8VUXGYAx7FNbVZBFU72rQw3GaCuG2AkrUQWnWTh5SpemP: '280',
                    j4Rh1cHtZFAQYGh7Y8RZwoXbkAPtZN46FmuYpKNiR3P2Dc2oz: '200',
                    j4RjraznxDKae1aGL2L2xzXPSf8qCjFbjuw9sPWkoiy1UqWCa: '160',
                    j4RuqkJ2Xqf3NTVRYBUqgbatKVZ31mbK59fWnq4ZzfZvhbhbN: '180',
                    j4RxTMa1QVucodYPfQGA2JrHxZP944dfJ8qdDDYKU4QbJCWNP: '140',
                    j4Rxkb1w9yB6WXroB2npKjRJJxwxbD8JjSQwMZFB31cf5aZAJ: '160',
                    j4RyLBbSUBvipuQLkjLyUGeFWEzmrnfYdpteDa2gYNoM13qEg: '160',
                    j4S998Thq5kQHyurofh8QfHrcFN2c1T19gTdMGUVVx5EHKgky: '220',
                  },
              },
```
Or is there a type error ?

Reply:
Yes, there comes an error due to the methods of type BN like `nmul()` or `toNumber()` used in custom hook.
Also the decorator seems not to return a observable, once I removed the `rxjs` operator `of`, an error occurs like `TypeError: observable?.subscribe is not a function`

Reply:
Got it. I'll give it a try on your branch maybe the mock decorator needs some fixes